WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -127,6 +127,29 @@ $venue-map-wrapper-height: 400px;
 }
 
 .competitions-table {
+  font-size: 12px;
```

Comment:
Can we get rid of this `font-size`? You did remove the `1em` already in https://github.com/cubing/worldcubeassociation.org/pull/330/files, yes?